app/src/main/java/com/example/tasky/agenda/data/local/model/AttendeeEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.tasky.agenda.data.local.model
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class AttendeeEntity(
+    @PrimaryKey
```

Comment:
Since you won´t generate the attendees ID, i´d add "autogenerate = false" to the PrimaryKey annotation

Same for event, task and reminder